# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/libtool/statics_at_usr_lib.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

if destdir is /lib, static lib could had been moved to /usr/lib by 
INSTALL_WRAPPER

 - Alejandro
--- ./ltmain.sh.orig	2003-09-19 12:43:07.000000000 -0400
+++ ./ltmain.sh	2003-09-19 12:44:25.000000000 -0400
@@ -5630,6 +5630,10 @@
       $show "$install_prog $file $oldlib"
       $run eval "$install_prog \$file \$oldlib" || exit $?
 
+      if [ "$destdir" == "/lib" -a ! -f "$oldlib" ]; then
+        oldlib="/usr/lib/$name"
+      fi
+
       if test -n "$stripme" && test -n "$striplib"; then
 	$show "$old_striplib $oldlib"
 	$run eval "$old_striplib $oldlib" || exit $?
--- ./libltdl/ltmain.sh.orig	2003-09-19 12:43:07.000000000 -0400
+++ ./libltdl/ltmain.sh	2003-09-19 12:44:25.000000000 -0400
@@ -5630,6 +5630,10 @@
       $show "$install_prog $file $oldlib"
       $run eval "$install_prog \$file \$oldlib" || exit $?
 
+      if [ "$destdir" == "/lib" -a ! -f "$oldlib" ]; then
+        oldlib="/usr/lib/$name"
+      fi
+
       if test -n "$stripme" && test -n "$striplib"; then
 	$show "$old_striplib $oldlib"
 	$run eval "$old_striplib $oldlib" || exit $?
